ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-R2-CUI-nonprod.json

```diff
@@ -479,6 +479,44 @@
       }
     ]
   },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "NOTIFY_LIP_RESPONDENT_CLAIMANT_CONFIRM_TO_PROCEED",
```

Comment:
aaa sorry, i am now confused does this event mean it notifies both respondent and claimant as it mentions both? perhaps if it only notifies defendant it should be NOTIFY_LIP_RESPONDENT_CONFIRM_TO_PROCEED?

Reply:
No problem... I explain to you... despite this one being Neeta's code, I used the same format to name the next one (notification for the applicant) because I understood that the intention is to specify that these two events happen when the CLAIMANT_CONFIRM_TO_PROCEED.

Reply:
ah i got it now. thanks Domini!